tests/test_orphaned.py

```
@@ -133,3 +136,191 @@ def test_multiple_extension_pattern(self):
+        dummy_file = source_dir / "orphaned.mkv"
+        dummy_file.write_text("dummy content")
+
+        orphaned_files = [str(dummy_file)]
+        recycle_bin = tmp_path / "recycle_bin"
+
+        delete_orphaned_files(orphaned_files, dry_run=True, client=mock_client, recycle_bin=str(recycle_bin))
+
+        assert dummy_file.exists()
+        assert "Would move to recycle bin (orphaned/uncategorized)" in caplog.text
+
+    def test_windows_path_handling(self, mock_client, tmp_path):
```

Ruff (0.14.5)

Unused method argument: `mock_client`

(ARG002)